app/controllers/HomeController.scala

```diff
@@ -118,17 +115,23 @@ class HomeController @Inject()(val homeView: views.html.Home,
       auditingService.extendedAudit(HomeAudit(user, paymentsDueMerged, overDuePaymentsCount, nextUpdatesTileViewModel))
```

Comment:
auditing expected to be very last step before rendering/sending html back to the client;
we might need apply design changes in the future in order to enforce this sequence of events;